samples/matter/light_bulb/boards/nrf54l15pdk_nrf54l15_cpuapp_0_3_0.overlay

```diff
@@ -6,10 +6,11 @@
 
 
 / {
-	aliases {
-		factory-data = &factory_data;
-		factory-data-memory-region = &cpuapp_rram;
+	chosen {
```

Comment:
I wonder whether it will work with 0.7.0 PDK revision. I think that we should remove the _0_3_0 suffix because now both 0.3.0 and 0.7.0 revisions use the default configuration.

Reply:
Ok, done.